ckanext/resource_type_validation/plugin.py

```diff
@@ -19,12 +19,20 @@ def configure(self, config):
 
     # IResourceController
 
+    # CKAN 2.9
```

Comment:
before_create is still used, in 2.10.

Reply:
https://docs.ckan.org/en/2.10/extensions/plugin-interfaces.html has no references to `before_create`.